Lib/test/test_decimal.py

```diff
@@ -4862,7 +4862,7 @@ def test_c_context_errors(self):
         for attr in ('prec', 'Emin', 'Emax', 'capitals', 'clamp'):
             self.assertRaises(OverflowError, setattr, c, attr, int_max+1)
             self.assertRaises(OverflowError, setattr, c, attr, -int_max-2)
-            if sys.platform != 'win32':
+            if not MS_WINDOWS:
```

Comment:
I'm not sure that this adds a lot of clarity.  Could you remove that?

Reply:
It is not about just this one file. The request was, as much as possible, to use constants in the tests rather than different approaches such as 'sys.platform.startswith("win")' and sys.platform == 'win32'. 

Compare with (if pasting images works)
![image](https://user-images.githubusercontent.com/10852597/58546671-24c7cf80-8206-11e9-9888-88221dc6e257.png)
 Otherwise:

Look at this file (Lib/test/_test_multiprocessing.py, https://github.com/python/cpython/pull/13648/files#diff-b046ab474480855fb4a01de88cfc82bb) - where, for windows both a local constant (WIN32) and sys.platform {==|!=) 'win32' is used.

imho - this adds clarity to the whole.